addons/l10n_de/models/base_document_layout.py

```diff
@@ -24,4 +24,4 @@ def _compute_l10n_de_template_data(self):
         ]
 
     def _compute_l10n_de_document_title(self):
-        self.l10n_de_document_title = 'Invoice'
+        self.l10n_de_document_title = _('Invoice')
```

Comment:
Can you please also add this in the `.pot` file?

Reply:
@william-andre done it~ thanks for checking!